### PR TITLE
Moved `_get_tag_ranges()` to kytos/core

### LINKED ISSUE #174: Move `_get_tag_ranges()` to `kytos/core`
Due to `vlan_range` epic,  `_get_tag_ranges()` is going to be use by `mef_eline` so it will need to be moved to `kytos/core`.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #174 

### Summary

Moved `_get_tag_ranges()` to `kytos/core/tag_ranges.py`
Now using `KytosInvalidRanges` exception
This PR relies in kytos PR [#428](https://github.com/kytos-ng/kytos/pull/428)

### Local tests

Unit tests are moved to `kytos/core`

### End-To-End Tests
```
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.3.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2, anyio-3.6.2
collected 244 items

tests/test_e2e_01_kytos_startup.py ..                                    [  0%]
tests/test_e2e_05_topology.py ..................                         [  8%]
tests/test_e2e_10_mef_eline.py ..........ss.....x.....x................  [ 24%]
tests/test_e2e_11_mef_eline.py ......                                    [ 27%]
tests/test_e2e_12_mef_eline.py .....Xx.                                  [ 30%]
tests/test_e2e_13_mef_eline.py ....Xs.s.....Xs.s.XXxX.xxxx..X........... [ 47%]
.                                                                        [ 47%]
tests/test_e2e_14_mef_eline.py x                                         [ 47%]
tests/test_e2e_15_mef_eline.py ....                                      [ 49%]
tests/test_e2e_20_flow_manager.py .....................                  [ 58%]
tests/test_e2e_21_flow_manager.py ...                                    [ 59%]
tests/test_e2e_22_flow_manager.py ...............                        [ 65%]
tests/test_e2e_23_flow_manager.py ..............                         [ 71%]
tests/test_e2e_30_of_lldp.py ....                                        [ 72%]
tests/test_e2e_31_of_lldp.py ...                                         [ 74%]
tests/test_e2e_32_of_lldp.py ...                                         [ 75%]
tests/test_e2e_40_sdntrace.py .............                              [ 80%]
tests/test_e2e_41_kytos_auth.py ........                                 [ 84%]
tests/test_e2e_42_sdntrace.py ..                                         [ 84%]
tests/test_e2e_50_maintenance.py ........................                [ 94%]
tests/test_e2e_60_of_multi_table.py .....                                [ 96%]
tests/test_e2e_70_kytos_stats.py ........                                [100%]
```

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
Nicely done extracting and reusing this part and introducing `KytosInvalidRanges`. 

Looks good, I'll approve this PR once the linked PR is also approved.

File: requirements/dev.in
Comment (viniarck) [inline requirements/dev.in:2]: Comment reminder to revert this pinned branch here once it's ready to land

File: main.py
Comment (viniarck) [inline main.py:508]: Cool to have `KytosTagError` here. Since all the child exceptions have great names and the `__repr__` is also including its name and the message, maybe in the detail you can just pass `str(err)`? Check out the current error below, since `"Error when setting tag ranges:"` is being concatenated it might get repetitive since `KytosSetTagRangeError` will be in the message, see what I mean, check this example below? Trivial stuff though.

Comment (Alopalao) [reply main.py:508]: Oh got it, lines 508 and 525 changed.

**Review by viniarck — APPROVED** [review]
Leaving this pre-approved. Let's revert the requirements/dev.in pinned branch once the related kytos PR lands, and then get this one here merged. Changelog updated for this one here isn't needed.

File: requirements/dev.in
Comment (Alopalao) [reply requirements/dev.in]: Reverted.

**Pull request merged**
